plugins/modules/ecs_service.py

```diff
@@ -89,6 +92,16 @@
         required: false
         type: bool
         default: false
+    deployment_controller:
+        description:
+          - The deployment controller to use for the service. If no deploymenet controller is specified, the ECS controller is used.
+        required: false
+        type: dict
+        suboptions:
+          type:
+            type: str
+            choices: ["ECS", "CODE_DEPLOY", "EXTERNAL"]
+            description: The deployment controller type to use.
```

Comment:
Could you please add a `version_added` key to both new options, to show which collection release it was added in?
The next release will be 1.5.0, in early April

You can see an example in ec2_vol here:
https://github.com/ansible-collections/amazon.aws/blob/main/plugins/modules/ec2_vol.py#L95-L107

Reply:
```suggestion
            description: The deployment controller type to use.
            version_added: 1.5.0
```